tools/maxquant/mqparam.py

```diff
@@ -94,11 +116,17 @@ def _make_exp_design(self, infiles):
                     if i in design:
                         index.append(i)
                     else:
-                        raise Exception("Invalid comlumn index in experimental"
+                        raise Exception("Invalid column index in experimental"
                                         + " design template: {}".format(i))
+                line_no = 0
                 for line in design_file:
```

Comment:
use the power of python Luke: http://book.pythontips.com/en/latest/enumerate.html